### ISSUE TITLE
How to add admin page?

### ISSUE THREAD

**Issue by aliscie** [tiangolo/fastapi#5080]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi_admin.app import app as admin_app
from fastapi import FastAPI

app = FastAPI()
app.mount("/admin", admin_app)
```

### Description

I tried that code but when i visit localhost:8000/admin i got 404 erorr is thier a sipmplified full project sample?

### Operating System

Linux, Windows, macOS

### Operating System Details

.

### FastAPI Version

Lates 

### Python Version

3.9

**Comment by JarroVGIT** [comment]
Seems like a question for that project?

**Comment by bfss** [comment]
Fastapi does not provide an admin page, seems you used another package fastapi_admin, you could look for help from there

**Comment by hbarnard** [comment]
Yes, I've been through this starlette admin provides a page: https://stackoverflow.com/questions/72813671/routing-with-mariadb-and-starlette-admin but I don't understand it fully and documentation is sparse.

It looks like I have to code out routes for every table in my DB if I use FastAPi? This was the underlying reason to try to use starlette admin...

**Comment by JarroVGIT** [comment]
Starlette-admin seems again another project. This is not a fastapi related question, please use the issues or discussions on those projects to resolve any questions related to those packages

**Comment by hbarnard** [comment]
Yes, however, Starlette underlies fastapi? So I felt that this small, one-off intervention would be useful. Mea culpa.

---------
https://www.hughbarnard.org
Twitter: @hughbarnard

------- Original Message -------

**Comment by mkdir700** [comment]
https://github.com/aminalaee/sqladmin

**Comment by acidjunk** [comment]
The FastAPI Admin project seems to have an excellent example:
https://github.com/fastapi-admin/fastapi-admin/tree/dev/examples

If that answers your question; it might be good to close it.

**Comment by tiangolo** [comment]
Thanks for the help here everyone! 🍰☕

Yep, if you haven't solved the problem, it's probably better to ask there.
